README.md

```diff
@@ -3,8 +3,6 @@ HDF5 version 1.15.0 currently under development
 ![HDF5 Logo](doxygen/img/HDF5.png)
 
 [![develop build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/main.yml?branch=develop&label=develop)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
-[![oneAPI CMake build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/intel-cmake.yml?branch=develop&label=oneAPI-C)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
-[![oneAPI Autotools build status](https://img.shields.io/github/actions/workflow/status/HDFGroup/hdf5/intel-auto.yml?branch=develop&label=oneAPI-A)](https://github.com/HDFGroup/hdf5/actions?query=branch%3Adevelop)
```

Comment:
Did you try adding "?event=workflow_call" to the branch and label options?

Reply:
@byrnHDF, No. Did you?

Reply:
okay, thanks for trying.